language/move-prover/bytecode-to-boogie/tests/goldenfiles/libra_account.bpl

```diff
@@ -1534,7 +1534,7 @@ ensures old(b#Boolean(Boolean(!(b#Boolean(ExistsResource(__m, LibraAccount_T_typ
     // declare local variables
     var __t2: Value; // AddressType()
     var __t3: Value; // LibraCoin_T_type_value()
-    var __t4: Value; // ByteArrayType()
+    var __t4: Value; // Vector_T_type_value(IntegerType())
```

Comment:
I guess this is unrelated to this change and just fixing a leftover problem from converting bytearray to vector<u8>, right?

Reply:
Good catch--I guess somewhat forget to regenerate the goldenfiles and CI didn't catch it (since it only runs on changes in the prover directory). Seems ok to fix it here.